config.py.sample

```diff
@@ -0,0 +1,19 @@
+import os
+
+
+DEBUG = True
+DEFAULT_LIMIT = 10
+DEFAULT_OFFSET = 0
+FROM_EMAIL = 'YOUR@EMAIL.COM'
+MAX_EMAIL_ADDRESS_SIZE = 256
+MAX_RETRIES = 3
+MAX_SUBJECT_SIZE = 256
```

Comment:
What if this app accepted attachments, how would you handle large attachments? (50MB+)